samples/grpc/restaurant-drone-deliveries-service-java/src/main/resources/persistence.conf

```diff
@@ -0,0 +1,37 @@
+akka.persistence.r2dbc.connection-factory = ${akka.persistence.r2dbc.postgres}
+akka {
+  persistence {
+    journal.plugin = "akka.persistence.r2dbc.journal"
+    snapshot-store.plugin = "akka.persistence.r2dbc.snapshot"
+    state.plugin = "akka.persistence.r2dbc.state"
+
+    r2dbc {
+      connection-factory {
+        host = "localhost"
+        host = ${?DB_HOST}
+        port = 5432
+        database = "postgres"
+        user = "postgres"
+        user = ${?DB_USER}
+        password = "postgres"
+        password = ${?DB_PASSWORD}
+      }
+
+      state = {
+        additional-columns {
+          "CentralDrone" = ["central.drones.LocationColumn"]
+        }
+      }
+
+      # FIXME disabled until we have fixed handling of FilteredEvents for publishing in r2dbc
+      journal.publish-events = off
```

Comment:
is this still a problem? but maybe we'll go over all FIXME in a separate scan

Reply:
Yes, I'll do a follow up work through taking care of FIXMEs